python/paddle/static/__init__.py

```diff
@@ -64,11 +64,14 @@
 
 from ..fluid.io import load_vars  # noqa: F401
 from ..fluid.io import save_vars  # noqa: F401
+from ..fluid.io import batch  # noqa: F401
 
 from ..fluid.layers import create_parameter  # noqa: F401
 from ..fluid.layers import create_global_var  # noqa: F401
 from ..fluid.layers.metric_op import auc  # noqa: F401
 from ..fluid.layers.metric_op import accuracy  # noqa: F401
+from ..fluid.contrib.layers import ctr_metric_bundle  # noqa: F401
+from ..fluid.layers import exponential_decay  # noqa: F401
 
 __all__ = [  #noqa
     'append_backward', 'gradients', 'Executor', 'global_scope', 'scope_guard',
```

Comment:
shall we add ctr_metric_bundle  and exponential_decay in list of \_\_all\_\_ ?

Reply:
done